chainstate/src/detail/chainstateref.rs

```diff
@@ -305,6 +305,13 @@ impl<'a, S: BlockchainStorageRead, O: OrphanBlocks> ChainstateRef<'a, S, O> {
         self.get_gen_block_index(&self.get_best_block_id()?)
     }
 
+    pub fn get_token_tx(
```

Comment:
`get_token_info`, and return that object we talked about in storage.

Reply:
Done